chargeio/service/Payment.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace chargeio
+{
+    public class Payment : IPayment
```

Comment:
Similar to my quesiton on IPayment: What is this for? It looks like dummy/placeholder code, and should be removed.